clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/AwsProvider.groovy

```diff
@@ -134,4 +130,34 @@ class AwsProvider extends AgentSchedulerAware implements SearchableProvider {
       return cacheView.getAll(INSTANCES.ns, possibleInstanceIdentifiers)*.id
     }
   }
+
+  // Eureka provider support
+  private Map<String, String> accounIdToAccountNameLookup = [:]
```

Comment:
I would kill this off and just make the supporting methods use AccountCredentialRepository. There are a small enough number of accounts in a system that linearly searching them for every call is not going to be significant. There is support for reloading the configuration that this would have to support otherwise